Podcast/InternalProfileViewController.swift

```diff
@@ -95,6 +95,8 @@ class InternalProfileViewController: ViewController, UITableViewDelegate, UITabl
         case (0,2): //subscriptions
             let subscriptionsViewController = SubscriptionsViewController()
             navigationController?.pushViewController(subscriptionsViewController, animated: true)
+        case (0,3): // shared content
+            navigationController?.pushViewController(SharedContentViewController(), animated: true)
```

Comment:
i feel like the shared content should go in an inbox from the feed, like how Instagram shows the DM inbox in the main feed instead of on profile?